Installation_TrueNAS_GUI.md

```diff
@@ -112,6 +112,106 @@ This is similar to installing Jellyfin but with fewer steps:
 
 - Start the jellyfin server
 
+## Hardware encoding (Intel)
+
+### Packages and script to add inside the Jail 
+
+- Back on the jails list find your newly created jail for jellyfin and click "Shell" :
+  - Execute Theses commands : 
+
+    ```
+    pkg install libva-utils
+    pkg install libva-intel-media-driver
+    ```
+
+  - Add lffmpeg script to add missing vaapi command
+
+     Create the lffmpeg script file (available here : [url](script/lffmpeg))
+
+     ```
+     cd /usr/local/bin
+     ee lffmpeg
+     ```
+
+     Paste the file content :
+
+     ```
+     #!/bin/sh
+     ffmpeg -hwaccel vaapi "$@"
+     ```
+
+     Close the editor with [ESC] and enter
+
+  - Make lffmpeg script executable
+
+     `chmod +x lffmpeg`
+
+### Script to add on the TrueNas Core Host
+
+- Loading kernel module + adding jail config to pass /dev/dri and /dev/drm
+  - Open a ssh shell on the TrueNas host 
+    Create a script file in the root folder (available here : [url](script/enable_gpu_jails.sh)) :
+
+    `ee /root/enable_gpu_jails.sh`
```

Comment:
Will this survive a reboot or OS upgrade on TrueNAS?

Reply:
Yes... however, the files are not exported with the configuration... so TrueNas team goes against theses options. As TrueNas Team rule we should put the file on a new Dataset on the Pool... but I feel it's okay to recreate a file if a nas has to be reinstaled.

Reply:
As long as it survives simple reboots and OS upgrades it should be fine for general users

Reply:
On my own server the file /root/scripts/plex-ruleset.sh (same thing with diffrent name) is present since july 2022... I've done all the updates from 12.0 to 13.0-U1... and never had problems.